Dockerfile

```diff
@@ -192,13 +192,17 @@ RUN BUILD_DEPS="build-essential \
     apt-get purge $BUILD_DEPS software-properties-common -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+ENV COMPOSER_HASH=${COMPOSER_HASH:-76a7060ccb93902cd7576b67264ad91c8a2700e2} \    
+    FITS_VERSION=${FITS_VERSION:-1.4.0}
+
 ## Let's go!  Finalize all remaining: djatoka, composer, drush, fits.
 RUN useradd --comment 'Islandora User' --no-create-home -d /var/www/html --system --uid $ISLANDORA_UID --user-group -s /bin/bash islandora && \
     chown -R islandora:www-data /var/www/html && \
     ## Temporary directory for composer, fits, etc...
     mkdir -p /tmp/build && \
+    cd /tmp/build/ && \
     ## COMPOSER
-    wget -O composer-setup.php https://raw.githubusercontent.com/composer/getcomposer.org/2091762d2ebef14c02301f3039c41d08468fb49e/web/installer && \
+    wget -O composer-setup.php https://raw.githubusercontent.com/composer/getcomposer.org/$COMPOSER_HASH/web/installer && \
```

Comment:
How does this work?  I'm not sure what it's doing here or what it's fetching.

Thanks!